scripts/promote-cherry-picks.ts

```diff
@@ -37,8 +45,52 @@ function getAmpVersionToCherrypick(
   return ampVersionToCherrypick.slice(-13);
 }
 
+async function getCherryPickedPRs(
+  ampVersion: string,
+  numberOfCherryPickedCommits: number
+): Promise<string[]> {
+  try {
+    const {data} = await octokit.rest.repos.listCommits({
+      owner: 'ampproject',
+      repo: 'amphtml',
+      sha: ampVersion,
+      per_page: numberOfCherryPickedCommits,
+    });
+    return data
+      .map(({commit}) => {
+        const [firstLine] = commit.message.split('\n');
+        const matches = firstLine?.match(/\(#(?<pullNumber>\d+)\)$/);
+        return matches?.groups?.pullNumber;
+      })
+      .filter(defined);
+  } catch (err) {
+    console.warn('Could not fetch the list of cherry picked PRs, skipping...');
+    console.warn('Exception thrown:', err);
+    return [];
+  }
+}
+
+function generateBody(
+  ampVersion: string,
+  channels: string,
+  cherryPickedPRs: string[]
+): string {
+  let body = `Promoting release ${ampVersion} to channels: ${channels}`;
+  if (cherryPickedPRs.length) {
+    body += '\n\nPRs included in this cherry pick:\n';
+    body += cherryPickedPRs
+      .map(
+        (pullNumber) =>
+          `* https://github.com/ampproject/amphtml/pull/${pullNumber}`
+      )
+      .join('\n');
+  }
+  body += `\n\n[Release branch commits](https://github.com/ampproject/amphtml/commits/amp-release-${ampVersion})`;
```

Comment:
Oops I missed this. Can the link say the branch name?

Reply:
You didn't miss this, I added it after your previous round of comments 😅
Done: 99f9b3d